spec/models/facility_spec.rb

```diff
@@ -185,11 +185,28 @@
   end
 
   describe "Validations" do
-    it { should validate_presence_of(:name) }
-    it { should validate_presence_of(:district) }
-    it { should validate_presence_of(:state) }
-    it { should validate_presence_of(:country) }
-    it { should validate_numericality_of(:pin) }
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:district) }
+    it { is_expected.to validate_presence_of(:state) }
+    it { is_expected.to validate_presence_of(:country) }
+    it { is_expected.to validate_numericality_of(:pin) }
+
+    describe "teleconsultation medical officers" do
+      context "when teleconsultation is enabled" do
+        subject { Facility.new(enable_teleconsultation: true) }
+
+        specify do
```

Comment:
Make this `it` as well?

Reply:
Oh yeah, good point. No reason for this one to be `specify`.